manifests/install.pp

```diff
@@ -1,15 +1,15 @@
 # Install the foreman proxy
 class foreman_proxy::install {
   if ! $foreman_proxy::custom_repo {
-    foreman::repos { 'foreman_proxy':
+    class { '::foreman::repo':
```

Comment:
> I now see that we do pull in EPEL and SCL as well

Configuring EPEL's correct, as in fact it's a dependency for the smart proxy too.  SCL isn't, but I'm not sure what we can do about that.



> I'm also worried about duplicate resource definitions since puppet-foreman also calls it as class { '::foreman::repo': }.

Good point, I'll change it to a regular include in puppet-foreman.